build.gradle

```diff
@@ -88,13 +88,17 @@ subprojects {
     }
 
     plugins.withId('java', { _ ->
-
         sourceSets {
             main.java.srcDirs = ["src"]
             main.resources.srcDirs = ["resources"]
             test.java.srcDirs = ["test"]
             test.resources.srcDirs = ["test-resources"]
         }
+
+        java {
+            sourceCompatibility = JavaVersion.VERSION_1_8
+            targetCompatibility = JavaVersion.VERSION_1_8
+        }
```

Comment:
Without this there is a warning that we may have misalignment on `javac` produced binaries and `kotlinc`